tests/e2e/tests/pilot/testdata/networking/v1alpha3/sidecar-scope-ns1-ns2.yaml

```diff
@@ -12,7 +12,7 @@ spec:
   egress:
   - port:
       number: 23145
-      protocol: TCP
+      protocol: http_proxy
```

Comment:
this seems very confusing given that http-proxy means something completely different in the "port name magic". Is this field not an enum? Why are we adding a new protocol?
also this looks like it has to be lower case